### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by crunchwrapdestroyer**

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -18,8 +26,13 @@ searchForm.addEventListener('submit', function (e) {
     //Look in the console to see what value we are getting
     console.log(inputVal)
     //Calls the searchBooks function with a argument of the users input value
-    searchBooks(inputVal)
```
Comment (EmpireAntz) [inline assets/js/index.js:21]: add this line back in and this should be good for merge

**Review by EmpireAntz — CHANGES_REQUESTED** [review]
line 21 for searchBooks(inputValue) just needs to be added back in

File: assets/js/index.js (hunk 1 above)
Comment (crunchwrapdestroyer) [reply assets/js/index.js:21]: Added!

File: index.html [hunk 2]
```diff
@@ -51,6 +54,16 @@ <h2>Find Your Favorite Book</h2>
             </div>
         </div>
     </div>
+
+    <div id="modal1" class="modal open" tabindex="0" style="z-index: 1003; display: none; opacity: 1; top: 10%; transform: scaleX(1) scaleY(1);">
+      <div class="modal-content">
+        <h4>Please enter a title, author, or </h4>
```
Comment (EmpireAntz) [inline index.html:60]: only thing I would change is have it just say "Please enter a book title!📚"

**Review by EmpireAntz — DISMISSED** [review]
This looks great! everything is functioning as it should and your code is clean and concise🎉😎

**Review by EmpireAntz — APPROVED** [review]
Looks Great Matt!

**Pull request merged**